vuepress/docs/next/tutorials/devops/manage-nginx.md

```diff
@@ -88,10 +88,10 @@ kind: ConfigMap
 
 Production environments may require additional common annotations:
 ```
-nginx.ingress.kubernetes.io/client-max-body-size: 200m # to upload client request (default is 1m)
-nginx.ingress.kubernetes.io/proxy-body-size: 200m # to upload large files (default is 10M)
-nginx.ingress.kubernetes.io/proxy-buffer-size: 64k # for the Keycloak auth-token (default is 16K)
-nginx.ingress.kubernetes.io/proxy-read-timeout: "600" # to increase the timeout when uploading large files
+nginx.ingress.kubernetes.io/client-max-body-size: 200m # to read client request (default is 1m)
```

Comment:
Description feels a bit off. Maybe "the maximum allowed size of the client request body" to match the nginx docs?

Reply:
@nshaw sure. i was trying to match the language for the proxy body size... should the proxy body size also be changed?

Reply:
@Lyd1aCla1r3 I think it's okay as-is. It's possible that one was replaced/deprecated but I can't find a definite answer on it - https://stackoverflow.com/questions/57509872/kubernetes-ingress-proxy-body-size-annotation-has-stopped-working